browser-test/src/support/admin_programs.ts

```diff
@@ -522,9 +522,29 @@ export class AdminPrograms {
   }
 
   async createNewVersion(programName: string) {
+    await this.createNewVersionWithFlag(programName, false)
+  }
+
+  async createNewVersionWithReadOnlyViewEnabled(programName: string) {
```

Comment:
**suggestion (non-blocking):** You don't need this method since it calls the next. It's just as easy for callers to do

`createNewVersion('program name', /* isReadOnlyView= */ true)`

as

`createNewVersionWithReadOnlyViewEnabled('program name')`